ping/ping_output.c

```diff
@@ -0,0 +1,230 @@
+/*
+ * SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2024-2025 Georg Pfuetzenreuter <mail+ip@georg-pfuetzenreuter.net>
```

Comment:
nit: We never bothered with updating copyright (now code changed a lot) for C code, I should have updated it. You base your code on _this_ file on existing iputils code (I would say at least from 2015 78aa5c1516d6f299331b4cd0efac5f0f53a28c0f, which was based on old code, some algorithms are older than 2006 when project was imported to git), IMHO there should be also iputils project copyright. Maybe them add below your copyright?
```
* Iputils project (c) 2014-2024
```

Reply:
I added

```
 * Copyright (c) 2014-2024 Iputils project
 * Copyright (c) 1989-2006 The Regents of the University of California
```

now, but I'm not sure who/what's between 2006 and 2014, would it classify as `Iputils project` as well? Or do you think the `Iputils project` line is enough? I'm not sure quite how old the algorithms the commit you mentioned bases on are.